src/components/Heading/Heading.tsx

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-interface HeadingProps {
+import { themeHelper, sxPropHelper, StyleProps } from '../../utils/styled';
+
+interface HeadingProps extends StyleProps {
     children: React.ReactNode;
     level: 1 | 2 | 3 | 4 | 5;
-    as?: React.ElementType;
 }
 
-const Heading = styled.h2<HeadingProps>((props) => ({
-    fontSize: (props.theme as any).fontSizes[props.level],
-    lineHeight: '125%',
-    fontFamily: 'Source Sans Pro, sans-serif',
-    fontWeight: 'bold',
-    color: (props.theme as any).colors.neutral[9],
-    '-webkit-font-smoothing': 'antialiased',
-    '-moz-osx-font-smoothing': 'grayscale',
-}));
+const Heading = styled.h2<HeadingProps>(
```

Comment:
What happens if you want a heading to be an `<h3>`? Swapping out a level prop will just change the prop size right?

Reply:
Yeah you could do
```
<Heading level={3} as={h3} />
```

Reply:
`as?: React.ElementType;` was removed from your prop definition...Do I just not know how styled components work? 😄

Reply:
`interface HeadingProps extends StyleProps`

Reply:
Neato-mosquito.